.kokoro/presubmit/doctest.cfg

```diff
@@ -0,0 +1,17 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Only run this nox session.
+env_vars: {
+    key: "NOX_SESSION"
+    value: "doctest"
+}
+
+env_vars: {
+    key: "GOOGLE_CLOUD_PROJECT"
+    value: "bigframes-load-testing"
```

Comment:
I suspect as the number of user journeys we want to test grows, it might make sense to test in a separate project. I'm especially worried about some APIs with limited per-project quota like Cloud Functions, BigQuery Remote Functions, and some Vertex + BQML flows. What do you think?

Reply:
I believe Cloud Support can increase almost any quota and some of them are meant to be increased (although not sure about Functions) but I agree it would make sense to share the burden bigframes-load-testing is doing right now for the sake of cleanliness if nothing else, since it has several different functions currently which could get confusing.